modules/search/instant-search/components/search-results.jsx

```diff
@@ -53,6 +55,13 @@ class SearchResults extends Component {
 					</p>
 				) }
 				{ results.map( result => this.render_result( result ) ) }
+				{ this.props.hasNextPage && (
+					<ScrollButton
+						enableAutoScrolling
```

Comment:
I don't think I understand this feature and looks like it isn't hooked up yet. Is this to scroll results to the top of the window when we load more?

Reply:
This property toggles the load-on-scroll behavior for `ScrollButton`. When this property is omitted or set to false, `ScrollButton` will no longer trigger its `onClick` handler.

Reply:
Ahhhh. Got it. Slick!